data/human/culture conversations.txt

```diff
@@ -103,6 +103,35 @@ mission "Trash Fire Crops"
 
 
 
+mission "Worker's Occasion"
+	minor
+	source
+		government "Republic"
+		attributes "dirt belt"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`Pausing to consider a restaurant for your dinner, you notice a local worker staring at the menu, hand shaking as he proceeds to walk in. His clothes are neatly worn, and look freshly brushed, but it's impossible not to notice the dirty marks and stains that just haven't washed out. The man is undeniably nervous and twitchy, yet no one kicks him out. Looking at the menu yourself, you realize the prices are very affordable for you but you recall how unreachable they would've been for you before your career change.`
```

Comment:
Does neatly worn here refer to wear and tear or how the man is wearing his clothes?

Reply:
How he's wearing them, the freshly brushed is their condition.

Reply:
I think I was just overanalyzing here, this doesn't need to be changed.